examples/hotrod/services/customer/database.go

```diff
@@ -75,9 +74,8 @@ func newDatabase(tracer trace.Tracer, logger log.Factory) *database {
 func (d *database) Get(ctx context.Context, customerID int) (*Customer, error) {
 	d.logger.For(ctx).Info("Loading customer", zap.Int("customer_id", customerID))
 
-	ctx, span := d.tracer.Start(ctx, "SQL SELECT", trace.WithSpanKind(trace.SpanKindClient))
+	_, span := d.tracer.Start(ctx, "SQL SELECT")
```

Comment:
don't follow the reasoning. Both ctx and span kind were appropriate

Reply:
What I mean is I don't agree with the change, it doesn't make sense to me. This particular code was already correct.

Reply:
Alright, I have tested the HotROD code by reverting the changes, and it looks as we expected.